src/app/cli/src/explore/api_server.rs

```diff
@@ -45,6 +51,23 @@ impl APIServer {
 
         let gql_schema = kamu_adapter_graphql::schema();
 
+        let login_credentials = accounts::PasswordLoginCredentials {
+            login: current_account_name.to_string(),
+            // Note: note a mistake, use identical login and password, equal to account name
+            password: current_account_name.to_string(),
+        };
+
+        let auth_svc = base_catalog.get_one::<dyn AuthenticationService>().unwrap();
+        let access_token = auth_svc
+            .login(
+                &accounts::LOGIN_METHOD_PASSWORD.to_string(),
+                serde_json::to_string::<accounts::PasswordLoginCredentials>(&login_credentials)
+                    .unwrap(),
+            )
+            .await
+            .unwrap()
```

Comment:
Convert these unwraps into `int_err()` once you move the logic into command.
The tool should not simply die with a stack trace, instead we have a mechanism of internal errors that map to CLI error handlers.